client/routes/workflow/components/workflow-graph/helpers/get-event-connections/activity-task-cancel-requested.js

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2020-2021 Uber Technologies Inc.
+//
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+const ActivityTaskCancelRequested = event => ({
```

Comment:
Wondering whether is a common practice to split things into so many small files in the front-end development. In general I think it is a good thing, but this looks like an overkill to me.

Reply:
This was previously in one big file (300 lines long or longer). We can put back into one file if you like. My thoughts on this was we could individually unit test each piece. Also searching for an existing EventType is easier because it is all alphabetical by default. Also adding a new event type is easier as you don't care about ordering as this is handled by the file system. I don't really mind either way.